src/lxc/af_unix.c

```diff
@@ -56,7 +56,7 @@ int lxc_abstract_unix_open(const char *path, int type, int flags)
 	addr.sun_family = AF_UNIX;
 
 	len = strlen(&path[1]) + 1;
-	if (len >= sizeof(addr.sun_path) - 1) {
```

Comment:
Yeah, I know you can give them names including \0 bytes but I'm always weary to mention this fact since I really don't want to deal with tracking down bugs that stem from a missing \0 byte at the end. :)